apps/api/src/app/portfolio/calculator/roai/portfolio-calculator-btcusd.spec.ts

```diff
@@ -108,6 +109,9 @@ describe('PortfolioCalculator', () => {
           name: 'Bitcoin',
           symbol: activity.symbol
         },
+        tags: activity.tags?.map((id) => {
```

Comment:
To avoid this as well as `parseDate()`, would it be cleaner to use the `class-transformer`? (dto to activity)